.github/workflows/release.yml

```diff
@@ -20,6 +20,12 @@ jobs:
       - name: Set the current release version
         id: release_version
         run: echo "release_version=${GITHUB_REF:11}" >> $GITHUB_OUTPUT
+      - name: projectVersion is not a snapshot
+        run: ./gradlew projectVersionNotSnapshot
+      - name: Micronaut Core is not a snapshot
+        run: ./gradlew micronautCoreNotSnapshot
+      - name: Micronaut Platform is not a snapshot
+        run: ./gradlew micronautPlatformNotSnapshot
```

Comment:
I'm probably wrong, but isn't it the pre-release action that sets these versions to non-snapshot versions?

So shouldn't these checks come after the `Run pre-release` step?

Reply:
`micronautCoreVersion` and `micronautPlatformVersion` are set manually. I move these checks after the `pre-release` step.

Reply:
Cool (projectVersion is not set manually I assume?)